### ISSUE TITLE
Editor: Allow editing wiki pages

### ISSUE THREAD

**Issue by ddpasa** [organicmaps/organicmaps#3646]
Please consider adding support for wiki and related tags in the editor.  This would include the following tags:

wikipedia
wikimedia_commons
wikidata

(wikidata tag isn't used by Organic Maps as far as I can see.  But it's a frequently used tag and some editors like ID automatically wikidata when the wikipedia link is added)

**Comment by endim8** [comment]
I'm not sure that it'd be a good idea to add this, especially with a lack of online autocomplete for values. Adding wiki tags is better done on https://www.openstreetmap.org/edit

**Comment by biodranik** [comment]
Adding Wiki tags is not something that is easily done by a general user in the fields, who may even not know about the OSM. Also, at least some basic validation is needed. I don't think that it is a good idea to enable it, at least for everyone. There was an issue about detecting professional OSMers and showing them "Expert" settings not visible to anyone else.

**Comment by ddpasa** [comment]
> Adding Wiki tags is not something that is easily done by a general user in the fields, who may even not know about the OSM. Also, at least some basic validation is needed. I don't think that it is a good idea to enable it, at least for everyone. There was an issue about detecting professional OSMers and showing them "Expert" settings not visible to anyone else.

Fair enough.  I would personally love this feature because lots of wiki tags are missing in my area and this is one of the features I use the most in OsmAnd.  That being said it is a rather an expert feature requiring care.

**Comment by simonschaufi** [comment]
The good thing about iD editor is that it auto detects wikidata. I guess there is a json file loaded which contains all the wikidata fields and when the name matches, the suggestion is shown to add it. Something like that would be awesome for OM.

**Comment by patepelo** [comment]
> (wikidata tag isn't used by Organic Maps as far as I can see)

Is it true that wikidata isn't used yet in OM as read mode-miscellaneous?
